src/microbit/e-Tekstil:-Tekstil-og-elektronikk!/README.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "eTekstil: Tekstil og elektronikk!"
+author: "Carl A. Myrland" 
+language: "nb"
+---
+
+
+# Lærerveiledning til eTekstil - innføring i tekstil og elektronikk {.intro}
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal elevene lage en valgfri filt-figur, montere LED-pærer på den, sy med strømførende sytråd og programmere mønster lysene skal blinke i, ved hjelp av BBC Micro:bit og blokkprogrammering i MakeCode. Oppgaven kan utvides og tilpasses alle aldersgrupper og erfaringsnivåer fra ca. 4. trinn og opp.
+
+## Oppgaven passer til: {.check}
+
+ __Fag__: Matematikk, kunst & håndverk, naturfag
+
+__Anbefalte trinn__: 4.-5. og oppover
+
+__Tema__: Innføring i tekstil og elektronikk - kreativ bruk av elektronikk sammen med fysiske produkter
+
+__Tidsbruk__: 90 min
+
+## Kompetansemål etter 4. årstrinn{.challenge}
```

Comment:
alle kompetansemål bør føres sammen, men markere årstrinn. se for eksempel https://oppgaver.kidsakoder.no/microbit/pxt_blinkende_hjerte/README med tilhørende markdown på https://raw.githubusercontent.com/kodeklubben/oppgaver/master/src/microbit/pxt_blinkende_hjerte/README.md